packages/cms-cli/commands/create.js

```diff
@@ -39,28 +44,28 @@ const createModule = (name, dest) => {
   }
   logger.log(`Creating ${destPath}`);
   fs.mkdirp(destPath);
-  logger.log(`Copying sample module files to ${destPath}`);
+  logger.log(`Creating module at ${destPath}`);
   fs.copySync(assetPath, destPath);
 };
 
-const createTemplate = (name, dest) => {
-  const assetPath = ASSET_PATHS.template;
+const createTemplate = (name, dest, type = 'template') => {
+  const assetPath = ASSET_PATHS[type];
   const filename = name.endsWith('.html') ? name : `${name}.html`;
   const filePath = path.join(dest, filename);
   if (fs.existsSync(filePath)) {
     logger.error(`The ${filePath} path already exists`);
     return;
   }
-  logger.log(`Making ${dest} if needed`);
+  logger.debug(`Making ${dest} if needed`);
   fs.mkdirp(dest);
-  logger.log(`Copying boilerplate template to ${filePath}`);
+  logger.log(`Creating file at ${filePath}`);
   fs.copySync(assetPath, filePath);
 };
 
 function configureCreateCommand(program) {
   program
     .version(version)
-    .description('Create assets from boilerplate.')
```

Comment:
For the basic assets like modules, templates, and partials, I feel like `boilerplate` is not the right term because the expectation is that there will need to be tons of modification. They are more like "code templates."